platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```diff
@@ -155,7 +156,13 @@ public T getClientForSubject(Subject subject) throws SecurityServiceException {
      * @see #getClientForSubject(Subject subject)
      */
     public WebClient getWebClientForSubject(Subject subject) throws SecurityServiceException {
-        return WebClient.fromClientObject(getClientForSubject(subject));
```

Comment:
I am not sure we need the fromClientObject since getClientForSubject always creates a new client.  A simple WebClient.client() call would work and then you do not need all of this new code.